### PR TITLE
[PAGOPA-1212] migrazione gha pagopa gps donation service

### PR DESCRIPTION
**Pull request by jacopocarlini**
depends on https://github.com/pagopa/github-actions-template/pull/26
[Release And Deploy Azure Function](https://github.com/pagopa/pagopa-gps-donation-service/actions/workflows/release_deploy.yml)

#### List of Changes

switch to GHA

#### How Has This Been Tested?

[Release And Deploy Azure Function](https://github.com/pagopa/pagopa-gps-donation-service/actions/workflows/release_deploy.yml)
#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+version: '3.8'
+
+services:
+  app:
+    container_name: 'gpd-donation'
```
Comment (cap-ang) [inline docker/docker-compose.yml]: maybe `gps` instead of `gpd` is more accurate?

**Review by cap-ang — CHANGES_REQUESTED** [review]
Great work 💯 
Before we can merge, we need to fix the code review pipeline.

File: helm/values-dev.yaml [hunk 2]
```diff
@@ -49,8 +50,8 @@ microservice-chart:
       cpu: "0.5"
   autoscaling:
     enable: true
-    minReplica: 3
-    maxReplica: 10
+    minReplica: 1
+    maxReplica: 2
```
Comment (pasqualespica) [inline helm/values-dev.yaml]: only 1
min & max

File: helm/values-uat.yaml [hunk 3]
```diff
@@ -49,8 +50,8 @@ microservice-chart:
       cpu: "0.5"
   autoscaling:
     enable: true
-    minReplica: 3
-    maxReplica: 10
+    minReplica: 1
+    maxReplica: 5
```
Comment (pasqualespica) [inline helm/values-uat.yaml]: ditto as for dev

**Review by pasqualespica — APPROVED** [review]

**Pull request merged**